File: app/src/main/java/com/OdiousPanda/thefweather/Model/AQI/City.java
package com.OdiousPanda.thefweather.Model.AQI;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class City {

    @SerializedName("geo")
    @Expose
    public List<Float> geo = null;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("url")
    @Expose
    public String url;

}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/aqi/City.kt
package com.odiousPanda.rainbowKt.model.dataSource.aqi


import com.google.gson.annotations.SerializedName

data class City(
    @SerializedName("geo")
    val geo: List<Double>,
    @SerializedName("name")
    val name: String,
    @SerializedName("url")
    val url: String
)